### DISCUSSION TITLE
FPS drop when connecting client to the Carla server

### DISCUSSION THREAD

**Discussion by Toufik98** [carla-simulator/carla#5203] [Q&A]
Hello to all, 

I am using Carla in one  of my projects. Indeed, I use the 0.9.13 version of Carla on Windows with an Intel Xeon processor and an NVIDIA QUADRO RTX 6000 GPU. 

When I run the server by itself, I have a 120 FPS rate, and it can go up to 180 FPS. But when I connect with a client, there is a sudden drop in FPS, and the server drops to 30 FPS, and when I run other clients it drops even more. 

Is this a Carla limitation, or is there a configuration to do to solve the problem? 

Thanks in advance for your replies.

**Comment by brscholz** [comment]
Hi, as soon as you connect a client, the server sends stuff via TCP, whether you receive and use it or not. I found that slows down the server massively. My GPU and CPU hang around and play cards while my localhost node is running hot...
